staging/src/k8s.io/apiserver/pkg/authentication/authenticator/interfaces.go

```diff
@@ -44,11 +45,11 @@ type Password interface {
 }
 
 // TokenFunc is a function that implements the Token interface.
-type TokenFunc func(token string) (user.Info, bool, error)
+type TokenFunc func(auds []string, token string) (user.Info, bool, error)
```

Comment:
need to spec the following:
* what does nil/empty auds mean?
* what should a token handler that is not audience aware do with auds?

Reply:
I added an explanation of the semantics on the authenticator.Token interface